NSS-Day-Cohort-66/rare-client-eager-miracles-1
# Ndw view profile

## Added userId primary key to local storage

### To Test

- Pull down `ndw-view-profile` in client directory AND make sure that you have the API `ndw-view-profile` pulled down as well 
- Make sure debugger is running in fetched branch
- Start react app with `npm start`
- Logout of app
- Log back in with a registered user
- Open dev tools
- Go to Application tab
- Open Local storage directory
- Make sure that user_id is in local storage and corresponds to the correct user
